apps/gitness/src/pages-v2/pull-request/pull-request-changes.tsx

```diff
@@ -0,0 +1,195 @@
+import * as Diff2Html from 'diff2html'
+import { atom, useAtom } from 'jotai'
+import { compact } from 'lodash-es'
+
+import {
+  EnumPullReqReviewDecision,
+  reviewSubmitPullReq,
+  useRawDiffQuery,
+  useReviewerListPullReqQuery
+} from '@harnessio/code-service-client'
+import { DiffViewerExchangeState, PullRequestChangesPage } from '@harnessio/ui/views'
+
+import { useAppContext } from '../../framework/context/AppContext'
+import { useGetRepoRef } from '../../framework/hooks/useGetRepoPath'
+import { useTranslationStore } from '../../i18n/stores/i18n-store'
+import { parseSpecificDiff } from '../../pages/pull-request/diff-utils'
+import { PullReqReviewDecision } from '../../pages/pull-request/types/types'
+import { changedFileId, DIFF2HTML_CONFIG } from '../../pages/pull-request/utils'
+import { PathParams } from '../../RouteDefinitions'
+import { normalizeGitRef } from '../../utils/git-utils'
+import { normalizeGitFilePath } from './pull-request-utils'
+import { usePullRequestProviderStore } from './stores/pull-request-provider-store'
+
+export const changesInfoAtom = atom<{ path?: string; raw?: string; fileViews?: Map<string, string> }>({})
+
+export default function PullRequestChanges() {
+  const { pullReqMetadata, refetchPullReq, refetchActivities, setDiffs } = usePullRequestProviderStore()
+  const { currentUser } = useAppContext()
+  const repoRef = useGetRepoRef()
+  const commitSHA = '' // TODO: when you implement commit filter will need commitSHA
+  const defaultCommitRange = compact(commitSHA?.split(/~1\.\.\.|\.\.\./g))
+  const [
+    commitRange
+    //  setCommitRange  TODO: add commit view filter dropdown to manage different commits
+  ] = useState(defaultCommitRange)
+  const [diffMode, setDiffMode] = useState<DiffModeEnum>(DiffModeEnum.Split)
+  const targetRef = useMemo(() => pullReqMetadata?.merge_base_sha, [pullReqMetadata?.merge_base_sha])
+  const sourceRef = useMemo(() => pullReqMetadata?.source_sha, [pullReqMetadata?.source_sha])
+  //   const [diffs, setDiffs] = useState<DiffFileEntry[]>()
+  const { pullRequestId } = useParams<PathParams>()
+  const prId = (pullRequestId && Number(pullRequestId)) || -1
+  const {
+    data: { body: reviewers } = {},
+    refetch: refetchReviewers,
+    isFetching: loadingReviewers
+  } = useReviewerListPullReqQuery({
+    repo_ref: repoRef,
+    pullreq_number: prId
+  })
+  const submitReview = useCallback(
+    (decision: PullReqReviewDecision) => {
+      reviewSubmitPullReq({
+        repo_ref: repoRef,
+        pullreq_number: prId,
+        body: { decision: decision as EnumPullReqReviewDecision, commit_sha: pullReqMetadata?.source_sha }
+      })
+        .then(() => {
+          // showSuccess(getString(decision === 'approved' ? 'pr.reviewSubmitted' : 'pr.requestSubmitted'))
+          refetchPullReq()
+          refetchReviewers()
+          refetchActivities()
+        })
+        .catch((exception: string) => {
+          console.warn(exception)
```

Comment:
Should this be shown on a toast instead?

Reply:
Not sure what the standard for using toast is atm. I can fix this in a follow up or when I clarify toast usage @vardanbansal-harness

Reply:
Got it